src/stores/analyticsContext.tsx

```diff
@@ -111,6 +115,20 @@ export const AnalyticsProvider = ({
     push(pushParams)
   }
 
+  // Track base location
+  const trackBaseLocation = (baseLocation: string): void => {
+    if (debug === true) {
+      console.debug(`ANALYTICS(trackBaseLocation):`, baseLocation)
+    }
+
+    // Track base location of user:
+    // setCustomDimension is required
+    // the 1 here corresponds to the id of the created custom dimension in the Matomo dashboard
+    // baseLocation comes from the personnel data api
+    push(['setCustomDimension', 1, baseLocation])
+    push(['trackPageView'])
```

Comment:
Question: Why are we pushing a page view here? I didn't see it in the page on [setting up a custom dimension](https://matomo.org/faq/reporting-tools/create-track-and-manage-custom-dimensions/#tracking-a-custom-dimension).

Reply:
I was referring to [here](https://developer.matomo.org/guides/tracking-javascript-guide#custom-dimensions) in the docs, under **Setting a custom dimension for the initial page view**. It was my understanding that this would only track the custom dimension on initial page load. The alternative would be that the custom dimension is set and then logged for each additional action _after_ the page loads. I hope that makes sense? I could be misinterpreting the docs. They aren't exactly thorough.

Reply:
Yeah it's unclear to me too. However, I do agree that tracking these on initial page load is what we want so I'm good with it. Thanks for the clarification.